src/sandwich-menu/components/sandwich-menu.jsx

```diff
@@ -0,0 +1,69 @@
+import MUIDataTable from 'mui-datatables'
+import React, {useEffect, useState} from "react";
+import {Box, Grid, Toolbar, Badge, IconButton} from "@material-ui/core";
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+import {useOnAddCart} from "../containers/hooks/use-on-add-cart";
+import {Title} from "../containers/details/title"
+import {ShowDetails} from "../containers/details/body";
+import {columns} from "../containers/table-columns";
+import {useDispatch, useSelector} from "react-redux";
+import {getAllSandwiches} from "../actions";
+
+export const Menu = () => {
+
+    const [selected, setSelected] = useState([]);
+    const dispatch = useDispatch()
+    useEffect(() => dispatch(getAllSandwiches()),[])
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/109502982-63a54c00-7aa2-11eb-8c76-ec4af39b8662.png)
Maybe this is what should happen in the `useInit` hook :)))

Reply:
Done